src/interface/gemm.hpp

```diff
@@ -19,12 +19,10 @@
  *
```

Comment:
Same as here can you call this gemm_interface.hpp as in the operation there are `operation/gemm.hpp` that may cause confusion when giving relative path

Reply:
Renamed to `interface/gemm_interface.hpp`